nodebuilder/core/grpc_client.go

```diff
@@ -0,0 +1,13 @@
+package core
+
+import (
+	"google.golang.org/grpc"
+)
+
+func NewGRPCClient(addr string, opts ...grpc.DialOption) (*grpc.ClientConn, error) {
+	conn, err := grpc.NewClient(addr, opts...)
+	if err != nil {
+		return nil, err
+	}
+	return conn, nil
+}
```

Comment:
Wonder if really need this proxy constructor. Seems useless now

Reply:
Oops I haven't pushed changes with removing the c-tor

Reply:
done